projects/core/src/user/user.module.ts

```diff
@@ -1,16 +0,0 @@
-import { ModuleWithProviders, NgModule } from '@angular/core';
-import { UserStoreModule } from './store/user-store.module';
-
-/**
- * @deprecated since 3.2, moved to the `@spartacus/user` package.
- */
-@NgModule({
-  imports: [UserStoreModule],
-})
-export class UserModule {
-  static forRoot(): ModuleWithProviders<UserModule> {
-    return {
-      ngModule: UserModule,
-    };
-  }
-}
```

Comment:
Can we document and provide migration this module removal?

Reply:
Done.